.env.development

```diff
@@ -41,3 +42,5 @@ FEATURE_ENABLE_AUTO_APPLIED_LICENSES='true'
 LEARNING_TYPE_FACET='true'
 LEARNER_SUPPORT_URL='https://support.edx.org/hc/en-us'
 FEATURE_BROWSE_AND_REQUEST='true'
+OPTIMIZELY_PROJECT_ID=1765060276
```

Comment:
Should this be in .env.development? Perhaps move it to a .env.private file for testing, if not?

Reply:
Yeah, we actually have a distinct "Dev" project in Optimizely where dev experiments can live.

It's not a secret, anyone could see the prod project ID if they viewed the page source of edx.org.

Reply:
@iloveagent57 I'm wondering more from an open-source point of view. It's a public repository, so it feels a bit strange to have our edX-specific project ID hardcoded here for anyone who happens to clone / run the application. For example, `frontend-app-learning` also uses the Optimizely script but [does not include `OPTIMIZELY_PROJECT_ID` in its .env files directly](https://github.com/openedx/frontend-app-learning/blob/c0c51a3028eeb25718a37f6cf6ce962dc6c59e5a/.env.development), solely relying on them coming from the edx-internal.